### PR TITLE
Notification if merged PR build fails

### PR DESCRIPTION
**Pull request by KaiSzuttor**
will add a python only after it seems working

### CODE REVIEW

File: src/core/CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 set(EspressoCore_SRC
+  abs.cpp
```
Comment (fweik) [inline src/core/CMakeLists.txt]: Is this related?

Comment (KaiSzuttor) [reply src/core/CMakeLists.txt]: ~yes its the newly introduced “anti blockading system” for particles~

Comment (KaiSzuttor) [reply src/core/CMakeLists.txt]: just for making the build fail

**Review by fweik — APPROVED** [review]

**Comment by mkuron** [comment]
Can this be adapted to also fix #2488?

**Comment by fweik** [comment]
Shouldn't that just work? The filter is only on the branch...

**Comment by mkuron** [comment]
Could you have it automatically close the issue if the build succeeds after someone manually presses the retry button in Gitlab CI?

**Comment by KaiSzuttor** [comment]
we dont expect that to happen very frequently, so I dont think this is necessary for now.

**Comment by hmenke** [comment]
I recommend using the [new issues API](https://gist.github.com/jonmagic/5282384165e0f86ef105) of GitHub instead of the regular REST API.  The former one is not rate-limited because it always requires authorization.

**Comment by KaiSzuttor** [comment]
I didn't get what the advantage is
